src/js/tweet-parser.js

```diff
@@ -47,13 +47,15 @@ const getTweetInfo = (tweet) => {
   const user = tweet.querySelector(usernameSelector) ? tweet.querySelector(usernameSelector).textContent : null;
   const text = tweet.querySelector(textSelector) ? tweet.querySelector(textSelector).textContent : null;
   const tweetid = tweet.querySelector(tweetIdSelector) ? tweet.querySelector(tweetIdSelector).href.match(/\d+\b/g)[0] : null;
+  const isHasBeenAnalyzed = false;
```

Comment:
I would change the variable name to `hasBeenAnalyzed` or `isAnalyzed` to follow with standard boolean variables naming conventions.